tests/schema.rs

```diff
@@ -518,3 +582,47 @@ fn ext_yang_data() {
         vec!["/ietf-restconf:restconf", "/ietf-restconf:restconf/data"]
     );
 }
+
+#[test]
+fn test_create_context_from_yang_library_path() {
+    let ctx = Context::new_from_yang_library_file(
+        YANG_LIBRARY_FILE,
+        DataFormat::JSON,
+        SEARCH_DIR,
+        ContextFlags::empty(),
+    )
+    .expect("Failed to create context");
+
+    let module_names = ctx
+        .modules(true)
+        .map(|m| m.name().to_string())
+        .collect::<HashSet<String>>();
+    let expected = HashSet::from([
```

Comment:
nit: ideally, you should use a `BTreeSet` instead, as the hashing algorithm from the standard library can change over time, which affects ordering and can cause this test to fail (I've had such issues in the past).

Reply:
You're right. In this case, you were comparing two HashSet instances, so any changes in internal ordering wouldn't affect the `Eq` comparison. I think I've been bitten a couple of times by the nondeterministic ordering of hash collections that my eyes pop whenever I see them. Sorry about this ^^